### PR TITLE
E loader pkg

### LINKED ISSUE #441: Merge e-loader-pkg-admin into libsys-webforms
Repo at http://github.com/sul-dlss/eloader-pkg-admin

### PR DESCRIPTION
**Pull request by jgreben**
Fixes #441

### CODE REVIEW

File: app/models/package.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+###
+#  Class to connect to the VND_PACKAGES table
+###
+class Package < ActiveRecord::Base
+  self.table_name = 'vnd_packages'
```
Comment (shelleydoljack) [inline app/models/package.rb:5]: I think the eloader vendor table is called vnd_packages_gen or something like that. Vnd_packages contains all the non eloader vendors.

File: Gemfile [hunk 2]
```diff
@@ -54,7 +54,7 @@ gem 'loofah', '~> 2.2.1'
 # sprockets ~> 3.7.2 Fixes know security vulnerability: CVE-2018-3760: Path Traversal in Sprockets
 gem 'sprockets', '~> 3.7.2'
 
-group :production do
+group :production, :development do
```
Comment (shelleydoljack) [inline Gemfile]: Not sure if we want this.

File: app/models/package.rb (hunk 1 above)
Comment (dlrueda) [reply app/models/package.rb:5]: vnd_packages_gen doesn't exist.  there is a vnd_files and vnd_hdrs used for non-eloader stuff.  (and there's a vnd_files_gen table).  but I think vnd_packages is right for eloader pkg admin

Comment (jgreben) [reply app/models/package.rb:5]: This is the rails schema:
```
create_table "vnd_packages", primary_key: "record_id", force: :cascade do |t|
    t.string   "package_id"
    t.string   "package_name"
    t.string   "package_status"
    t.string   "data_pickup_type"
    t.string   "afs_path"
    t.string   "ftp_server"
    t.string   "ftp_user"
    t.string   "ftp_password"
    t.string   "ftp_directory"
    t.string   "ftp_file_prefix"
    t.string   "ftp_list_type"
    t.string   "package_url"
    t.datetime "date_entered"
    t.string   "vendor_name"
    t.string   "holding_code"
    t.string   "comments"
    t.datetime "date_modified"
    t.string   "put_file_loc"
    t.string   "afs_search_string"
    t.string   "url_field"
    t.string   "vendor_id_read"
    t.string   "vendor_id_write"
    t.string   "access_note"
    t.string   "export_note"
    t.string   "junktag_file"
    t.string   "encoding_level"
    t.string   "vnd_catcode"
    t.string   "match_opts"
    t.string   "proc_type"
    t.string   "update_040"
    t.string   "rpt_mail"
    t.string   "access_urls_plats"
    t.string   "date_cat"
    t.string   "export_auth"
    t.string   "preprocess_modify_script"
    t.string   "preprocess_split_script"
    t.string   "preprocess_put_script"
    t.datetime "created_at",               null: false
    t.datetime "updated_at",               null: false
```

and this is the oracle schema:
screen shot 2018-08-27 at 4 24 04 pm

Comment (shelleydoljack) [reply app/models/package.rb:5]: You’re right. Sorry! I was going off of my memory. I think I was confusing it with the table that holds the files.

**Pull request merged**